source/faq.txt

```diff
@@ -232,4 +234,11 @@ Can I customize chunk distributions when syncing into a sharded cluster?
 No, you can't configure ``mongosync`` to customize chunk distributions
 on a destination sharded cluster.  ``mongosync`` samples each collection
 during initialization to determine how to distribute documents
-efficiently across the destination cluster’s shards after migration.
\ No newline at end of file
+efficiently across the destination cluster’s shards after migration.
+
+Can I use ``mongosync`` to set up a Disaster Recovery cluster?
```

Comment:
on second thought, it's better to say "maintain" instead of "set up" here, since the key issue is the lack of an ability to keep the cluster up to date with the source